templates/envs/gnosis/consensus/lighthouse.tmpl

```diff
@@ -0,0 +1,14 @@
+{{/* lighthouse.tmpl */}}
+{{ define "consensus" }}
+# --- Consensus Layer - Beacon Node - configuration ---
+CC_PEER_COUNT=50
+CC_LOG_LEVEL=info
+EC_API_URL={{.ExecutionApiURL}}
+EC_AUTH_URL={{.ExecutionAuthURL}}
+CC_INSTANCE_NAME=Lighthouse
+CC_IMAGE_VERSION={{if .CcImage }}{{.CcImage}}{{else}}sigp/lighthouse{{end}}
+CC_DATA_DIR={{.CcDataDir}}
+CC_JWT_SECRET_PATH={{.JWTSecretPath}}
+CL_FEE_RECIPIENT={{.FeeRecipient}}
```

Comment:
Please add the CL_BOOTNODES var for lighthouse template.

Reply:
Done